!SESSION 2019-05-17 15:39:28.763 -----------------------------------------------
eclipse.buildId=Release 14.6 Build SDK_P.68d
java.version=1.6.0_21
java.vendor=Sun Microsystems Inc.
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=sr
Command-line arguments:  -data D:\ra126-2016\Bomberman -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.cdt.core 1 0 2019-05-17 15:41:58.001
!MESSAGE Indexed 'boomberman_app' (5 sources, 41 headers) in 0.61 sec: 2,189 declarations; 3,009 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0.00%)

!ENTRY org.eclipse.cdt.core 1 0 2019-05-17 15:41:58.319
!MESSAGE Indexed 'bomberman_app1' (5 sources, 39 headers) in 0.31 sec: 2,161 declarations; 2,514 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0.00%)

!ENTRY org.eclipse.cdt.core 1 0 2019-05-17 15:41:58.650
!MESSAGE Indexed 'batle_city_bsp_0' (30 sources, 26 headers) in 0.33 sec: 718 declarations; 3,454 references; 5 unresolved inclusions; 0 syntax errors; 58 unresolved names (1.37%)

!ENTRY org.eclipse.cdt.core 1 0 2019-05-17 15:41:58.885
!MESSAGE Indexed 'Boomberman' (5 sources, 31 headers) in 0.23 sec: 2,015 declarations; 2,205 references; 5 unresolved inclusions; 0 syntax errors; 6 unresolved names (0.14%)

!ENTRY org.eclipse.core.jobs 4 2 2019-05-17 16:13:58.194
!MESSAGE An internal error occurred during: "Launching boomberman_app.elf".
!STACK 0
java.lang.RuntimeException: No Target with ID 0 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(Unknown Source)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(Unknown Source)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2019-05-17 16:18:23.710
!MESSAGE An internal error occurred during: "Launching boomberman_app.elf".
!STACK 0
java.lang.RuntimeException: No Target with ID 0 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(Unknown Source)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(Unknown Source)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2019-05-17 16:20:13.686
!MESSAGE An internal error occurred during: "Launching boomberman_app.elf".
!STACK 0
java.lang.RuntimeException: No Target with ID 0 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(Unknown Source)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(Unknown Source)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.core.jobs 4 2 2019-05-17 16:25:02.495
!MESSAGE An internal error occurred during: "Launching boomberman_app.elf".
!STACK 0
java.lang.RuntimeException: No Target with ID 0 in the System


	at com.xilinx.sdk.targetmanager.internal.TM.stop(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(Unknown Source)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at com.xilinx.sdk.debug.ui.XilinxAppLaunchDelegateWrapper.launch(Unknown Source)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:855)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:704)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1047)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1251)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)

!ENTRY org.eclipse.debug.core 4 0 2019-05-17 16:40:37.032
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Exception in org.eclipse.debug.core.DebugPlugin.stop() of bundle org.eclipse.debug.core.
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.stop(BundleContextImpl.java:791)
	at org.eclipse.osgi.framework.internal.core.BundleHost.stopWorker(BundleHost.java:510)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.suspend(AbstractBundle.java:566)
	at org.eclipse.osgi.framework.internal.core.Framework.suspendBundle(Framework.java:1206)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.decFWSL(StartLevelManager.java:592)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.doSetStartLevel(StartLevelManager.java:257)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.shutdown(StartLevelManager.java:215)
	at org.eclipse.osgi.framework.internal.core.InternalSystemBundle.suspend(InternalSystemBundle.java:284)
	at org.eclipse.osgi.framework.internal.core.Framework.shutdown(Framework.java:692)
	at org.eclipse.osgi.framework.internal.core.Framework.close(Framework.java:600)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.shutdown(EclipseStarter.java:399)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:629)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:584)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1438)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1414)
Caused by: java.lang.NullPointerException
	at com.xilinx.sdk.targetmanager.internal.json.XMDJSONProcess.issueCommand(Unknown Source)
	at com.xilinx.sdk.targetmanager.internal.TM.stop(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(Unknown Source)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at org.eclipse.debug.internal.core.LaunchManager.shutdown(LaunchManager.java:2359)
	at org.eclipse.debug.core.DebugPlugin.stop(DebugPlugin.java:622)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl$2.run(BundleContextImpl.java:771)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.stop(BundleContextImpl.java:764)
	... 19 more
Root exception:
java.lang.NullPointerException
	at com.xilinx.sdk.targetmanager.internal.json.XMDJSONProcess.issueCommand(Unknown Source)
	at com.xilinx.sdk.targetmanager.internal.TM.stop(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.AppRunner.stopRun(Unknown Source)
	at com.xilinx.sdk.debug.core.internal.RemoteProcess.terminate(Unknown Source)
	at org.eclipse.debug.core.Launch.terminate(Launch.java:252)
	at org.eclipse.debug.internal.core.LaunchManager.shutdown(LaunchManager.java:2359)
	at org.eclipse.debug.core.DebugPlugin.stop(DebugPlugin.java:622)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl$2.run(BundleContextImpl.java:771)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.framework.internal.core.BundleContextImpl.stop(BundleContextImpl.java:764)
	at org.eclipse.osgi.framework.internal.core.BundleHost.stopWorker(BundleHost.java:510)
	at org.eclipse.osgi.framework.internal.core.AbstractBundle.suspend(AbstractBundle.java:566)
	at org.eclipse.osgi.framework.internal.core.Framework.suspendBundle(Framework.java:1206)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.decFWSL(StartLevelManager.java:592)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.doSetStartLevel(StartLevelManager.java:257)
	at org.eclipse.osgi.framework.internal.core.StartLevelManager.shutdown(StartLevelManager.java:215)
	at org.eclipse.osgi.framework.internal.core.InternalSystemBundle.suspend(InternalSystemBundle.java:284)
	at org.eclipse.osgi.framework.internal.core.Framework.shutdown(Framework.java:692)
	at org.eclipse.osgi.framework.internal.core.Framework.close(Framework.java:600)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.shutdown(EclipseStarter.java:399)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:629)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:584)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1438)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1414)
